### DISCUSSION TITLE
Cant overwrite conflict between font-size and leading

### DISCUSSION THREAD

**Discussion by phoenixbart** [dcastil/tailwind-merge#401] [Help]
Maybe I'm missing something, but I can't overwrite the conflict between the font size and leading styling. At https://github.com/dcastil/tailwind-merge/blob/v2.2.2/docs/api-reference.md#extendtailwindmerge I see that you can override this conflict with this code:
```
override: {
        conflictingClassGroups: {
            // ↓ ID of class group which creates a conflict with…
            //           ↓ …classes from groups with these IDs
            //   Here we remove the default conflict between the font-size and leading class
            //   groups.
            'font-size': [],
        },
```

But is doesn't seem to work. Is this a bug?
Can someone help me?

**Comment by phoenixbart** [comment]
To reproduce the issue io made a very basic playground:
https://playcode.io/1833749
here you see the leading class is removed, even with this tw-merge config.

**Reply by dcastil** [reply]
Hey @phoenixbart! 👋

Thanks for the reproduction example. Just saw that you're using the key `overwrite` key in there, but it needs to be `override` instead. When I changed the key, the behavior was correct again.

**Reply by phoenixbart** [reply]
Hi dcastil,
Ah, stupid of me for not seeing that. Thank you ! :D

I have still one similar question: In a project we are still using an older version of tw-merge: v1.14.0.
Is it possible to also remove the conflict between font size and leading there?
I updated the playground using this version and config.
https://playcode.io/1833749

**Reply by dcastil** [reply]
Haha no worries. 😁

Yes, you can also do it in v1.14.0! It's just a bit more code:

```js
const twMerge = extendTailwindMerge(
    { /* your config */ },
    (config) => {
        delete config.conflictingClassGroups['font-size']
        return config
    },
)
```

**Reply by phoenixbart** [reply]
Yeah that worked! Thank you so much :D
You make my day ^.^
